lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -164,6 +341,3830 @@ module Resources
+            'dosageInstruction': [
+              {
+                'text': '100 mg = 1 cap, Oral, BID, 20 cap, 0 Refill(s)',
+                'patientInstruction': '1 Capsules Oral (given by mouth) 2 times a day. Refills: 0.',
+                'timing': {
+                  'repeat': {
+                    'boundsPeriod': {
+                      'start': '2020-05-11T10:15:00.000-05:00'
+                    }
+                  },
+                  'code': {
+                    'coding': [
+                      {
+                        'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003',
+                        'code': '696528',
+                        'display': 'BID'
+                      },
+                      {
+                        'system': 'http://terminology.hl7.org/CodeSystem/v3-GTSAbbreviation',
+                        'code': 'BID',
+                        'display': 'BID'
+                      },
+                      {
+                        'system': 'http://snomed.info/sct',
+                        'code': '229799001',
+                        'display': 'Twice a day (qualifier value)'
+                      }
+                    ],
+                    'text': 'BID'
+                  }
+                },
+                'route': {
+                  'coding': [
+                    {
+                      'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4001',
+                      'code': '318185',
+                      'display': 'Oral',
+                      'userSelected': true
+                    },
+                    {
+                      'system': 'http://snomed.info/sct',
+                      'code': '26643006',
+                      'display': 'Oral route (qualifier value)',
+                      'userSelected': false
+                    }
+                  ],
+                  'text': 'Oral'
+                },
+                'doseAndRate': [
+                  {
+                    'doseQuantity': {
+                      'value': 1,
+                      'unit': 'cap',
+                      'system': 'http://unitsofmeasure.org',
+                      'code': '{Capsule}'
+                    }
+                  }
+                ]
+              }
+            ],
+            'dispenseRequest': {
+              'validityPeriod': {
+                'start': '2020-05-11T10:15:20.000-05:00'
+              },
+              'numberOfRepeatsAllowed': 0,
+              'quantity': {
+                'value': 20,
+                'unit': 'cap',
+                'system': 'http://unitsofmeasure.org',
+                'code': '{Capsule}'
+              }
+            },
+            'substitution': {
+              'allowedBoolean': true
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314974793',
```

Comment:
These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Reply:
comment addressed